libcobblersignatures/__init__.py

```diff
@@ -2,3 +2,221 @@
 This library contains additionally a CLI which enables you to have a feature equal way to manipulate the objects, as
 well as to im- and export them.
 """
+
+import json
+import urllib.request
+from typing import List, Union
+
+from libcobblersignatures.enums import ExportTypes, ImportTypes
+from libcobblersignatures.models.osbreed import OsBreed
+from libcobblersignatures.models.osversion import Osversion
+
+
+class Signatures:
+    """
+    This is the entry point of the library. You may create one or more objects of this class. Each instance of it is
+    independent of the other. I recommend only to have on object at a time of this class. This library doesn't persist
+    anything except you explicitly use the export method of the instance.
+    """
+
+    _rootkey: str
+    _signaturesjson: dict
+    _osbreeds: List[OsBreed]
+
+    def __init__(self):
+        """
+        If you want to create an empty file just use this constructor. If you want to import one use this and then run
+        an import function.
+        """
+        self._rootkey = "breeds"
+        self._signaturesjson = {}
+        self._osbreeds = []
+
+    @property
+    def signaturesjson(self) -> dict:
+        """
+        This property represents the json which was im- or exported.
+
+        :raises JSONDecodeError: If invalid json is given
+        :return: The last valid content of the json which the library knows about.
+        """
+        return self._signaturesjson
+
+    @signaturesjson.setter
+    def signaturesjson(self, value: str):
+        """
+        The setter for the exported data structure.
+
+        :param value: The str which will be loaded by the Python json utility.
+        """
+        if value is None:
+            pass
```

Comment:
This isn't doing anything at the moment. Should that be a `return None` or is that a placeholder?

Reply:
No indeed that should have been a return statement. I will fix that.